app/api/probe_dock/reports_api.rb

```diff
@@ -59,7 +59,7 @@ def serialization_options reports
       get do
         authorize! TestReport, :index
 
-        rel = policy_scope(TestReport).order 'created_at DESC'
+        rel = policy_scope(TestReport).order 'test_reports.created_at DESC'
```

Comment:
Add parentheses to method call. Also on lines 64, 76, 113, 115 and probably elsewhere in this method.